app/controllers/notes_controller.rb

```diff
@@ -28,15 +28,20 @@ def create
   # Retrieve a Note given its :id
   def show
     @activities = @note.activities.latest
+    load_conflicting_versions(@note)
   end
 
   def edit
+    # TODO what if there are no versions? E.g. legacy data
```

Comment:
Wouldn't PaperTrail make sure `#versions` always returns a `[]` even for legacy instances?